google/cloud/storage/tests/testbench.py

```diff
@@ -469,6 +485,21 @@ def update_acl(self, entity, role):
 gcs.debug = True
 
 
+# TODO(#821) TODO(#820) - until we implement CreateBucket + DeleteBucket insert
+# a well-known bucket based on the BUCKET_NAME environment (this is set by our
+# CI builds), to make the integration tests easy to write.
+def insert_magic_bucket(base_url):
+    if len(GCS_BUCKETS) == 0:
+        bucket_name = os.environ.get('BUCKET_NAME', 'test-bucket')
+        bucket = GcsBucket(base_url, bucket_name)
+        # Get the metageneration to 4, the value expected by the integration
+        # tests.
+        bucket.update_from_metadata({})
```

Comment:
Here and below - what's the purpose of these 3 repeated lines?

Reply:
What the comment says "get the metageneration to 4".  Maybe we should change the tests, but I thought we should do that later.

Reply:
Ahhh. Right. I can read, I promise :P

It totally didn't register in my head that doing an empty update will bump the metageneration.